lib/oidc/options/OAuthOptionsConstructor.ts

```diff
@@ -48,7 +48,7 @@ function assertValidConfig(args) {
       'Required usage: new OktaAuth({issuer: "https://{yourOktaDomain}.com/oauth2/{authServerId}"})');
   }
 
-  if (issuer.indexOf('-admin.') !== -1) {
+  if (issuer.indexOf('-admin.okta') !== -1) {
```

Comment:
Question, is it worth it to maintain a list of all okta domains to perform the search against? 

It's probably a low chance of this but thinking about someone having a domain like `auth-admin.otkamanium.com` 😅

Reply:
If no value add in that this LGTM!